dj/typing.py

```diff
@@ -79,6 +80,13 @@ def process_array_args(*args: str) -> Tuple["ColumnType"]:
     """
     Validate the args of an ARRAY
     """
+    if len(args) > 1:
```

Comment:
Perhaps `if len(args) != 1:` to cover a missing arg too? Although the message would probably need to change.

Reply:
Good idea! I added an altered error message for the missing arg case: `ARRAY needs subtype defined`.